python/hail/context.py

```diff
@@ -500,6 +500,10 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
         :param bool skip_bad_ad: If True, set AD FORMAT field with
             wrong number of elements to missing, rather than setting
             the entire genotype to missing.
+
+        :return: A VariantDataset read from the VCF file
```

Comment:
In import Plink, read, and import gen, we say it just like this. Wanted to be consistent with that.